chpl/chpl-service/src/main/java/gov/healthit/chpl/scheduler/presenter/Sed2015CsvPresenter.java

```diff
@@ -31,25 +31,20 @@
      */
     public int presentAsFile(final File file, final List<CertifiedProductSearchDetails> cpList) {
         int numRows = 0;
-        OutputStreamWriter writer = null;
-        CSVPrinter csvPrinter = null;
         try {
-            writer = new OutputStreamWriter(new FileOutputStream(file), StandardCharsets.UTF_8);
-            csvPrinter = new CSVPrinter(writer, CSVFormat.EXCEL);
-
-            csvPrinter.printRecord(generateHeaderValues());
-
-            for (CertifiedProductSearchDetails currListing : cpList) {
-                List<List<String>> rows = generateRows(currListing);
-                if (rows != null) { // can return null to skip a row
-                    for(List<String> row : rows) {
-                        csvPrinter.printRecord(row);
-                        numRows++;
+            try (OutputStreamWriter writer = new OutputStreamWriter(new FileOutputStream(file), StandardCharsets.UTF_8);
```

Comment:
Wait, is this a `try { try () { } }` structure? With nothing in the first try?

Reply:
I had a mental fart...  For some reason, it never dawned on me that that try() could have a catch.  Not sure what I was thinking.  
Fixed.